### PR TITLE
underwater fix

### LINKED ISSUE #7021: Underwater sea tiles block clicks
I'm sure everyone knows about this but hoo boy was it painful watching a popular SS13 youtuber Randolph P Checkers (or well, popular by our standards) struggle with this: https://www.youtube.com/watch?v=cCkwtjupD-0

Is there anything we can do to make it so the sea can be 'clicked through'?

### PR DESCRIPTION
**Pull request by alexkar598**
:cl: alexkar598
fix: you can now click on things underwater
 /:cl:

fixes #7021 

first pr on paradise

### CODE REVIEW

**Comment by Crazylemon64** [comment]
👀 github editor

😦

**Comment by alexkar598** [comment]
i copied pasted from dream maker

**Comment by alexkar598** [comment]
i have git for windows but i cant get it to work without redownloading the whole codebase

**Comment by Crazylemon64** [comment]
https://github.com/alexkar598/Paradise/blob/c34301c7481ad0742bfde424dba998fbb68ea371/code/game/turfs/unsimulated/beach.dm#L9

Add `mouse_opacity = 0` to the statement where the overlay is created above.

**Comment by alexkar598** [comment]
mouse_opacity = 0 makes it so you CANT click on the floor

**Comment by alexkar598** [comment]
sand and dense types of water are untouched

**Comment by alexkar598** [comment]
0 // transparent to mouse
1 // opaque where icon is also opaque
2 // completely opaque

**Comment by alexkar598** [comment]
Oh I see what you mean but I've tried and it gave me a runtime

**Comment by alexkar598** [comment]
It's because the overlay is an extension of the floor. And it's not really a problem since there is no use at clicking water

**Comment by Fox-McCloud** [comment]
@Crazylemon64 that doesn't work, unfortunately.

It really comes down to do you want the tile to be unclickable, normally or be clickable but have weird alt-click behaviors.

File: code/game/turfs/unsimulated/beach.dm [hunk 1]
```diff
@@ -89,14 +94,17 @@
 	name = "Sunken Floor"
 	icon = 'icons/turf/floors.dmi'
 	icon_state = "wood"
+	mouse_opacity = 0
```
Comment (Fox-McCloud) [inline code/game/turfs/unsimulated/beach.dm]: Redundant.

File: code/game/turfs/unsimulated/beach.dm [hunk 2]
```diff
@@ -89,14 +94,17 @@
 	name = "Sunken Floor"
 	icon = 'icons/turf/floors.dmi'
 	icon_state = "wood"
+	mouse_opacity = 0
 
 /turf/unsimulated/beach/water/deep/sand_floor
 	name = "Sea Floor"
 	icon_state = "sand"
+	mouse_opacity = 0
```
Comment (Fox-McCloud) [inline code/game/turfs/unsimulated/beach.dm]: Redundant.

File: code/game/turfs/simulated/floor/misc_floor.dm [hunk 3]
```diff
@@ -38,6 +38,7 @@
 /turf/simulated/floor/beach
 	name = "beach"
 	icon = 'icons/misc/beach.dmi'
+	mouse_opacity = 0
```
Comment (Fox-McCloud) [inline code/game/turfs/simulated/floor/misc_floor.dm]: These don't have water on them.

**Comment by alexkar598** [comment]
@Fox-McCloud is that better?

File: code/game/turfs/simulated/floor/misc_floor.dm [hunk 4]
```diff
@@ -47,10 +47,12 @@
 	name = "coastline"
 	icon = 'icons/misc/beach2.dmi'
 	icon_state = "sandwater"
+	mouse_opacity = 0
```
Comment (Fox-McCloud) [inline code/game/turfs/simulated/floor/misc_floor.dm]: This doesn't have a water overlay on it.

**Comment by alexkar598** [comment]
@Fox-McCloud fixed

**Comment by Crazylemon64** [comment]
Tested - it works, and it should do

**Pull request merged**